docs/dom-testing-library/api-events.md

```diff
@@ -85,8 +85,8 @@ createEvent[eventName](node: HTMLElement, eventProperties: Object)
 
 Convenience methods for creating DOM events that can then be fired by
 `fireEvent`, allowing you to have a reference to the event created: this might
-be useful if you need to access event properties that cannot be initiated
-programmatically (such as `timeStamp`).
+be useful to access event properties that cannot be initiated programmatically
```

Comment:
Not sure I understand the removal of "if you need" and similar statements throughout. What rule is causing that?

Reply:
It is due to `textlint-rule-stop-words`

source: https://github.com/sapegin/textlint-rule-stop-words/blob/master/dict.txt

Reply:
Hmm, can we hold off on that rule for now? The "if you need/want" phrases seem to be the only things getting picked up by the rule so far and the usages would need to be rephrased a bit more carefully than what autocorrect implied.

cc @kentcdodds

Reply:
Notice that only the rules with "`>`" are autofixed. The other ones remain as warnings (check out the screenshot on the PR description).

But of course no problem on taking it back